### PR TITLE
fix(editor3): Fix the save of last change in editor3

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

File: scripts/core/editor3/directive.js [hunk 1]
```diff
@@ -59,17 +59,21 @@ class Editor3Directive {
 
             /**
              * @type {Object}
-             * @description A JSON object representing the Content State of the Draft
-             * editor. When available, it is used to show content, using `convertFromRaw`.
-             * Either this, or value have to be set. Use this for most accurate behavior.
+             * @description Item which is being edited
              */
-            editorState: '=?',
```
Comment (petrjasek) [inline scripts/core/editor3/directive.js]: won't this break annotations?

File: scripts/apps/authoring/views/article-edit.html [hunk 2]
```diff
@@ -520,7 +521,8 @@
         data-item="item.extra[field._id]" data-html="true" data-limit="schema[field._id].maxlength"></span>
     <div sd-editor3
         data-single-line="field.field_options.single"
-        data-value="item.extra[field._id]"
```
Comment (petrjasek) [inline scripts/apps/authoring/views/article-edit.html]: this didn't work for assignment?

File: scripts/core/editor3/store/index.js [hunk 3]
```diff
@@ -79,14 +86,20 @@ function onChange(content) {
  * use available HTML. If none are available, an empty ContentState is created.
  */
 function getInitialContent(props) {
-    // we have an editor state stored in the DB
-    if (props.editorState && typeof props.editorState === 'object') {
-        return applyInlineStyles(convertFromRaw(props.editorState)).contentState;
+    // we have an editor state stored in the DB for body field
+    if (props.field === 'body_html' && props.item.editor_state &&
```
Comment (petrjasek) [inline scripts/core/editor3/store/index.js]: why do it here and make editor do some field specific behavior?

**Comment by petrjasek** [comment]
pls add some description what is the issue you've found

File: scripts/apps/authoring/views/article-edit.html (hunk 2 above)
Comment (ioanpocol) [reply scripts/apps/authoring/views/article-edit.html]: The problem was that on angular on value was not updated with the last html from editor3. So I sent the item and the field name and the html is initialized/saved from/to item.

File: scripts/core/editor3/store/index.js (hunk 3 above)
Comment (ioanpocol) [reply scripts/core/editor3/store/index.js]: The editor_state is used only for body_html. We can add a new parameter to specify if the editor_state is used or not.

**Comment by ioanpocol** [comment]
As PR title specified the last change in editor3 was never saved. This problem appear because of some synchronize problem of directive bidirectional parameters.

File: scripts/core/editor3/store/index.js (hunk 3 above)
Comment (petrjasek) [reply scripts/core/editor3/store/index.js]: ok so we could use the old editorState param and only set it for body_html right?

Comment (ioanpocol) [reply scripts/core/editor3/store/index.js]: I will use a boolean parameter for this.

**Pull request merged**